### DISCUSSION TITLE
[ASK] RX Only Device

### DISCUSSION THREAD

**Discussion by handiko** [peterantypas/maiana#108] [Q&A]
Hi, I want to build this project as a receive-only AIS device and send the nmea sentence to pc over a usb-to-serial cable. 
My question is, do I still need the GPS module and the EEPROM chip?
Thanks

**Comment by peterantypas** [comment]
Probably not, but firmware may need to be tweaked.

**Reply by handiko** [reply]
> 

Hi Peter,
After booting, does the firmware check the availability of the EEPROM and GPS, and if the EEPROM and GPS are not available then it just stops, or it will just straight into receive mode and spits out the AIVDM to the uart?
Thanks

**Reply by peterantypas** [reply]
It should be fine without GPS, but I never tested without an EEPROM.

**Reply by handiko** [reply]
Thank you.

**Reply by handiko** [reply]
Hi, again Peter.
In the folder latest/firmware/Transponder, there are firmware Lxx-xxx-Breakout and Lxx-xxx-BreakoutBootloader. What is the difference?
Which firmware do I need to flash into the MCU of the RF module?
Thanks

**Reply by peterantypas** [reply]
Ideally, first you build and install the bootloader, then use one of the Python scripts in the bootloader project to install a new firmware binary over serial (one that is linked for bootloader). The non-bootloader binaries are old and will not be updated.

**Reply by handiko** [reply]
The non-bootloader binaries are located at /latest/Firmware/Transponder/Binaries, right?

**Reply by peterantypas** [reply]
No, there are no "non-bootloader" binaries in the repository. I updated the README in that folder a few days ago. You will have to build from the Eclipse project yourself.

**Reply by handiko** [reply]
Hi again, Peter.
I hope you don't mind me asking too much.
I have a problem running the python scripts you mentioned before on my Ubuntu laptop. I can't successfully install wxPyhton that it requires. Are these scripts only meant to be running from a raspberry pi?
thanks.
